muskathlon/templates/muskathlon_my_home.xml

```diff
@@ -50,7 +28,8 @@
                     </t>
                     <div id="modal_flight_details" class="modal fade cms_modal_form" tabindex="-1" role="dialog"
                          aria-labelledby="donation_modal_label" aria-hidden="true">
-                        <div class="modal-dialog modal-xl" role="document">
+                        <!-- TODO Replace by modal-xl when Bootstrap 4.2 is supported by Odoo -->
+                        <div class="modal-dialog modal-lg" role="document">
```

Comment:
Do we really rollback this change ? Don't we use bootstrap 4.2 now ?

Reply:
Yes I checked and Odoo 12 uses bootstrap 4.1.3 unfortunately :cry:

Reply:
Ok, too bad